helper/include/odp/helper/macros.h

```diff
@@ -56,6 +56,27 @@ extern "C" {
 		abs_v < 0 ? -abs_v : abs_v;	\
 	})
 
+/**
+ * Return division of two numbers rounded up to the closest integer (positive
+ * values expected)
+ */
+#define ODPH_ROUNDUP_DIV(numer, denom)			\
+	__extension__ ({				\
+		__typeof__(numer) _numer = (numer);	\
+		__typeof__(denom) _denom = (denom);	\
+		(_numer + _denom - 1) / _denom;		\
+	})
+
+/**
+ * Round up 'x' to alignment 'align' (positive values expected)
+ */
+#define ODPH_ROUNDUP_ALIGN(x, align)				\
```

Comment:
Could be named `ODPH_ALIGN()`. This macro could also fit into the align API module. Should we limit `align` values to powers of two for efficiency?

Reply:
About power-of-two alignment, probably need then another new macro, application can choose whichever is suitable.

Reply:
Changed as discussed offline in v2.